### ISSUE TITLE
Failure with latest version of e2b-code-interpreter

### ISSUE THREAD

**Issue by cbornet** [datastax/astra-assistants-api#75]
There was a breaking change in `e2b-code-interpreter`: https://github.com/e2b-dev/code-interpreter/commit/92ce0268d6c47b6a9ede5c7f785fa022c6c88b42
And astra-assistant doesn't constrain the `e2b-code-interpreter` version in its manifest.

Suggestion for a fix:
add `e2b-code-interpreter={version="^0.0.10", optional=true}` to `[tool.poetry.dependencies]`

**Comment by phact** [comment]
Thanks will do

**Comment by cbornet** [comment]
No, the problem is that this dependency is only in an optional group.
So it's not put in the manifest when building the package.
Only the dependencies that are in `[tool.poetry.dependencies]` get to the manifest.

**Comment by cbornet** [comment]
Currently the package METADATA file contains
```
etadata-Version: 2.1
Name: astra-assistants
Version: 2.2.2
Summary: Astra Assistants API - drop in replacement for OpenAI Assistants, powered by AstraDB
Author: phact
Author-email: estevezsebastian@gmail.com
Requires-Python: >=3.10,<4.0
Classifier: Programming Language :: Python :: 3
Classifier: Programming Language :: Python :: 3.10
Classifier: Programming Language :: Python :: 3.11
Classifier: Programming Language :: Python :: 3.12
Requires-Dist: aiohttp (>=3.9.4,<4.0.0)
Requires-Dist: boto3 (>=1.34.31,<2.0.0)
Requires-Dist: httpx (>=0.27.0,<0.28.0)
Requires-Dist: litellm (>=1.36.0,<2.0.0)
Requires-Dist: lsprotocol (>=2023.0.1,<2024.0.0)
Requires-Dist: openai (>=1.20.0,<2.0.0)
Requires-Dist: python-dotenv (>=1.0.1,<2.0.0)
Requires-Dist: python-lsp-jsonrpc (>=1.1.2,<2.0.0)
Requires-Dist: ruff (>=0.6.2,<0.7.0)
Requires-Dist: tree-sitter (>=0.23.0,<0.24.0)
Requires-Dist: tree-sitter-python (>=0.23.0,<0.24.0)
Description-Content-Type: text/markdown
```
So there's nothing regarding the supported `e2b-code-interpreter` version.
You may have to define an extra to get an optional dep in there.

**Comment by cbornet** [comment]
And it would be good to put a constraint on `e2b` too as `e2b-code-interpreter` doesn't do it...

**Comment by phact** [comment]
Thanks! Fixed in `v2.2.5`
